### ISSUE TITLE
Static Website explaining the project

### ISSUE THREAD

**Issue by DhairyaBahl** [hash-define-organization/Hash-Whiteboard#19]
### Description
A static website explaining how to use this whiteboard and it should also have the section of guide to contribute and other important stuff like that.  You can choose docusaurus or just gatsby itself and create a website. Jekyll will also work for this. Temporarily we can also manage with normal HTML CSS.

### Contribution Tip
Due to the large size of this issue, contributors can make PRs and even create issues they are facing in this repo and each issue will be treated as Level3 because static website is on top priority.

Most importantly you should know the ins and outs of this project ! Then only you will be able to work on this issue. Since you will be explaining those ins and outs in the website itself.

### Must include parts in website
* Contributors Guide
* How to build/run project in your local
* Pre-requisites before starting contributing to this project.

**Comment by DhairyaBahl** [comment]
@vedant-jain03 I strongly suggest you to create another branch for this purpose ! Like `DOCS-Main` and choose the tech stack for that docs wisely as per the no. of contributors available for that.

**Comment by VaibhavUpreti** [comment]
I think we should prefer NextJS as those are really easy to make with some basic tailwindCSS ,although making this would take a lot of time, I would like to work on this by posting my inputs and progress daily over here.

**Comment by vedant-jain03** [comment]
@DhairyaBahl What I feel is NextJS will be really overwhelming for the project. 
I have an idea ,

- We can make project multisite with navbar having the option
  - Home (Landing page, Mission, Open source programs, footer)
  - About Us (Description, Contributors)
  - Drawing board (Main)
  - Usage or Guide (For Every Function there will be description respectively)
- We can use react-router-dom to have the functionality of routing in react
- Contributors can easily contribute as this will be in the scope of frontend and react.
- For contributors we can just add Navbar(without design), Home Page component, About us component, Guide component. Let the contributors work on design and implementation.

@DhairyaBahl WDYT?

**Comment by MitAbhay** [comment]
@vedant-jain03  I agree with you,  you should make different components so that we the contributors can contribute more efficiently. We can add all readme content in the Guide component.

**Comment by ithiel883** [comment]
can we use VuejS instead?

**Comment by vedant-jain03** [comment]
> can we use VuejS instead?

Any specific reason why?

**Comment by DhairyaBahl** [comment]
> ree with you, you should make different components so that we the contributors can contribute more efficiently. We can add all readme content in the G

I think all of your listed ideas are great and should be implemented as soon as possible ! Kindly create appropriate issues for each one of them. Great Job.

**Comment by DhairyaBahl** [comment]
@vedant-jain03 How about using [this](https://github.com/shuding/nextra) for this purpose. Idea of this project is great. Checkout if this fits our requirements.

**Comment by vedant-jain03** [comment]
Sorry for the late reply,

@DhairyaBahl I think it may be great if we have a lot of functions or a detailed explanation of a function. We can definitely consider the UI of the site you refer but what I feel is making all the forums or docs for the initial version of the project is not appropriate until and unless we have some really great function to work on.

What we can do :
- I will set react-routing for About Us, Home page, and Use Guide. Also a simple navbar to route to them.
- We can make issues of levels as follows:
   - Design About Us(Level 2)
   - Design Responsive navbar(Level -2)
   - Design Home page(Level -2)
   - Design Footer(Level -1)
   - Design User Guide(Level -3) 

Let me know your views then I will start the procedure!

**Comment by mayank-del** [comment]
@DhairyaBahl  ,i am a gssoc 22  participant, can i contribute in this issue.

**Comment by ithiel883** [comment]
> > can we use VuejS instead?
> 
> Any specific reason why?

cause it's better

**Comment by navyasharma0203** [comment]
Can I work on this??

**Comment by vedant-jain03** [comment]
> Can I work on this??

This issue is under discussion, can you check out other open & unassigned issues!
